docker/jupyter/Dockerfile

```diff
@@ -257,6 +257,15 @@ RUN R -e 'install.packages(c( \
  && echo "R_LIBS=/home/jupyter-user/.rpackages" > /home/jupyter-user/.Renviron \
  && chown -R $USER:users /home/jupyter-user/.rpackages
 
+#######################
+# JupyterLab Extension Requirements
+#######################
+
+RUN apt-get update \
+ && apt-get install -t testing -yq --no-install-recommends \
+    nodejs \
```

Comment:
Maybe we could just include `nodejs` and `npm` in the python 3 section? Adding a new layer increases the image size & build time.